crates/pessimistic-proof/src/local_exit_tree/mod.rs

```diff
@@ -106,6 +107,41 @@ where
 
         root
     }
+
+    /// Returns true if `other_root` is the root of a LET whose first
+    /// `self.leaf_count` leaves are the same as the leaves of `self`.
+    /// `next_leaf` is the leaf at index `self.leaf_count` in the other tree and
+    /// `proof` is the Merkle proof of `next_leaf`.
+    pub fn is_subtree(
```

Comment:
If I read the description correctly, this could be called `is_subtree_of` to make it a bit clearer which way around the subtree relation goes.

Reply:
Also, the presence of `next_leaf` suggests this checks strict subset/subtree. May be worth mentioning in the comment